sdk/compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -1646,6 +1661,80 @@ execDocTest opts scriptDar (ImportSource importSource) files =
       withDamlIdeState opts logger diagnosticsLogger $ \ideState ->
           docTest ideState files'
 
+data MultiPackageManifestEntry = MultiPackageManifestEntry
+  { packageName :: String
+  , packageVersion :: String
+  , packageDir :: FilePath
+  , packageSrc :: FilePath
+  , packageFileHashes :: Map.Map FilePath String
+  , packageHash :: String
+  , output :: FilePath
+  , packageDeps :: [String]
+  , darDeps :: [FilePath]
+  }
+  deriving (Show, Eq, Generic, ToJSON, FromJSON)
+
+-- Map can't be a bifunctor because of this `Ord` constraint, but its a shame such a function is defined in Data.Map
```

Comment:
Did you cherry pick all of the commits from 18832? I notice that this comment is still the erroneous one, which was fixed in https://github.com/digital-asset/daml/pull/18832/commits/2f55d5a28012f8e7001fd4d93e3f0ce6e44d94b9

Reply:
resolved